File: OnlineShopAPI/AddressMicroservice/Address.Api/Models/AddAddressRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Address.Api.Models
{
    public class AddAddressRequest
    {
        public long AddressId { get; set; }

        public string CustomerName { get; set; }

        public string UnitStreet { get; set; }

        public string City { get; set; }

        public string State { get; set; }

        public string Zipcode { get; set; }

        public long AccountId { get; set; }
    }
}

File: OnlineShopUI/OnlineShop/src/app/model/requests/add-address-request.ts
export class AddAddressRequest {
    addressId: number;
    customerName: string;
    unitStreet: string;
    city: string;
    state: string;
    zipcode: string;
    accountId: number;

    constructor(addressId: number, customerName: string, unitStreet: string, city: string,
        state: string, zipcode: string, accountId: number){
        this.addressId = addressId;
        this.customerName = customerName;
        this.unitStreet = unitStreet;
        this.city = city;
        this.state = state;
        this.zipcode = zipcode;
        this.accountId = accountId;
    }
}
